filters/auth/oidc_introspection.go

```diff
@@ -42,6 +42,14 @@ func NewOIDCQueryClaimsFilter() filters.Spec {
 	}
 }
 
+// Sets OIDC claims in the state bag.
+// Intended for use with the oidcClaimsQuery filter.
```

Comment:
Hm, the filter also uses sub claim from the container https://github.com/zalando/skipper/blob/44139f3a87d0fb455d9564af5207546f25ee622a/filters/auth/oidc_introspection.go#L123-L136

Reply:
See also related https://github.com/zalando/skipper/pull/3287

Reply:
I see. I can add a check and set the sub key to empty if it's not a string value.

Reply:
I'm not sure why this filter is the one setting the auth-user key (via authorized) though. I feel like it should be up to whatever filter obtained the credentials to begin with.